acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/privacy/condition/ExpectValidPrivateTransactionReceipt.java

```diff
@@ -43,15 +43,7 @@ public void verify(final PrivacyNode node) {
     assertThat(actualReceipt)
         .usingRecursiveComparison()
         .ignoringFields(
-            "commitmentHash",
-            "logs",
-            "blockHash",
-            "blockNumber",
-            "logsBloom",
-            "transactionIndex") // TODO: The fields blockHash, blockNumber, logsBloom and
-        // transactionIndex have to be ignored as the class
-        // org.web3j.protocol.besu.response.privacy.PrivateTransactionReceipt does not contain these
-        // fields. Once web3j has been updated these ignores can be removed.
+            "commitmentHash", "logs", "blockHash", "blockNumber", "logsBloom", "transactionIndex")
```

Comment:
Have you checked if we have the fields being returned by web3j? If we don't, it might be a good idea to keep the previous comment explaining why we are ignoring these fields.

Reply:
Done